src/server/models/listItem/listItemEvent.ts

```diff
@@ -122,14 +123,27 @@ export const EVENTS = {
     },
   }),
 
-  [ListItemEvent.REMINDER]: (updatedJsonData = {}, reference?: string): EventCreate<"REMINDER"> => ({
-    type: ListItemEvent.REMINDER,
-    jsonData: {
-      eventName: "reminder",
-      updatedJsonData,
-      ...{ reference },
-    },
-  }),
+  [ListItemEvent.REMINDER]: (
+    response: SendEmailResponse,
```

Comment:
cc @alisalaman-cyb 

I've change this so the response from govuk notify is a "first class"..property? `event.jsonData.response` will be the (trimmed) response from notify. it will include the template details, so template id and version number, as well as the id that notify returns. in future it can be used for a lookup to check status (only applicable if we add "failed" events). but mainly template so you can check if the right email has been sent.